db/models.py

```diff
@@ -50,3 +57,46 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
The `Order` model references the `User` model using `settings.AUTH_USER_MODEL`. Since the `User` model is defined in the same file, it should be referenced directly as `User` instead of using `settings.AUTH_USER_MODEL`.